web_server/authors/templates/posted.html

```diff
@@ -0,0 +1,21 @@
+{% load static %}
+<!DOCTYPE html>
```

Comment:
For consistency, could you make your pages derive from the index.html? This means you only need to specify the content of the page rather than the entire html and head each time. If this solution is not applicable, we need some way to make the pages appear consistent.